plugins/fclrave/CMakeLists.txt

```diff
@@ -82,7 +83,7 @@ if( FCL_FOUND )
     )
     target_link_libraries(fclrave PRIVATE boost_assertion_failed PUBLIC libopenrave ${FCL_LIBRARIES})
     # ${FCL_CFLAGS_OTHER} is useless as CMAKE_CXX_STANDARD now requires 14
-    set_target_properties(fclrave PROPERTIES COMPILE_FLAGS "${PLUGIN_COMPILE_FLAGS}" LINK_FLAGS "${PLUGIN_LINK_FLAGS} ${FCL_LDFLAGS}")
+    set_target_properties(fclrave PROPERTIES COMPILE_FLAGS "${PLUGIN_COMPILE_FLAGS}" LINK_FLAGS "${PLUGIN_LINK_FLAGS} ${FCL_LDFLAGS_STR}")
```

Comment:
My script uses ninja, so #640 is required. It is good time to revisit it, isn't it?